LCD Interfacing/LCD.c

```diff
@@ -0,0 +1,95 @@
+#include "LCD.h"
+
+void lcd_send_nibble(uint8_t ui8RS, uint8_t ui8Val)
+{
+	/* is the value cmd or data*/
+	if( ui8RS == 0 )
+		LCD_COMMAND_PINS->FIOCLR |= BS(LCD_RS);
+	else
+		LCD_COMMAND_PINS->FIOSET |= BS(LCD_RS);
+	
+	/* write data */
+	LCD_COMMAND_PINS->FIOCLR |= BS(LCD_RW);
+	// write nibble on D4 to D7 pins
+	LCD_DATA_PINS->FIOCLR = (BS(LCD_D7)|BS(LCD_D6)|BS(LCD_D5)|BS(LCD_D4));
+//		LCD_DATA_PINS->FIOCLR = 0x000000f0; //Alternate Before sending data, clear pins

```

Comment:
Why commented code included in PR? If required to be optionally used we can have a conditional compilation to include the latter part.

Reply:
Removed the alternate method of code. We could in future having such conditional compilation.